reana/cli.py

```diff
@@ -1054,13 +1054,13 @@ def git_push(component):  # noqa: D301
 
 @click.option('--user', '-u', default='reanahub',
               help='DockerHub user name [reanahub]')
-@click.option('--tag', '-t', default='',
+@click.option('--tag', '-t', default='latest',
```

Comment:
Beware, this would not build `latest` tag but rather a different one for each build as described by dirty....